examples/torchaudio/README.md

```diff
@@ -1,23 +1,36 @@
+# Serve Audio transcription built with Torch Audio using Lightning Triton Serve
+
 ### Step 1
 
-Clone this repo and cd to examples/torchaudio
+Install lightning using
+
+```bash
+pip install -U lightning
+```
 
 ### Step 2
 
-Create a new env and install lightning
+Clone this repo and cd to examples/torchaudio
+
+```bash
+git clone https://github.com/Lightning-AI/LAI-Triton-Server-Component.git
+cd LAI-Triton-Server-Component/examples/torchaudio
+```
 
 ### Step 3
 
-run 
+run the component using below command (if you need to run it in clud, add `--cloud` at the end of the command) 
 
 ```bash
 lightning run app serve_torch_audio.py --setup
 ```
 
 ### Step 4
 
-Once the server is up, run
+Once the server is up and running, you should see the client code in the Lightning App UI.
```

Comment:
people wont know what a "Lightning App UI" is
let's say:
"Once the server is running, Lightning will open a new tab in your browser. To access your server you can use the code snippet shown in that tab or the pre-packaged [`client.py` ](examples/torchaudio/client.py)